packages/common/package.json

```diff
@@ -45,8 +45,8 @@
     "@osn/provider-options": "^1.0.2"
   },
   "peerDependencies": {
-    "@polkadot/api": "^7.9.1",
-    "@polkadot/extension-dapp": "^0.43.1",
+    "@polkadot/api": "^8.5.1",
```

Comment:
is this update confirmed to be fine? 
some times we get break changes while updating this

Reply:
confirmed version by @wliyongfeng